src/providers/osso.js

```diff
@@ -0,0 +1,20 @@
+export default (options) => {
+  return {
+    id: 'osso',
+    name: 'SAML SSO',
```

Comment:
can i read this from options? seems like the best approach. we want to be mroe white labelled, so "SAML SSO" feels most appropriate, but i'd want the implementer to be able to decide on this label.

does this work?

```suggestion
    name: options.label || 'SAML SSO',
```

Reply:
I think best just to do `name: 'SAML SSO'` as matches convention for the other providers.

If `name` is specified in options then it will automatically override the value. 👍

Reply:
neat!